### ISSUE TITLE
Max Temperature Limit

### ISSUE THREAD

**Issue by cesarlopes** [nicehash/NiceHashQuickMiner#435]
I was studying the documentation of other miner and I found an interesting feature that is the maximum temperature limit, when a video card reaches that value the miner stop mining for that card for a few minutes.

I think it would be super interesting to implement a similar feature in NiceHash QuickMiner, where we could define a maximum temperature where the video card would stop mining for a few minutes if it reaches that value, would be an extra layer of security.

**Comment by nicehashdev** [comment]
This is already available: https://github.com/nicehash/NiceHashQuickMiner/wiki/Smart-Fan-Mode-Target-plus-decrease-MCC

**Comment by cesarlopes** [comment]
It is not exactly this feature that I suggested, today for example I like to set a fixed value for the fan, for example 50% in the 1660 and 60% in my 3070, they make less noise here by my side and the temperatures do not exceed 54 degrees .

However, if I restart the machine, or the excavator itself, sometimes this temperature ends up rising and because I have set the fixed mode of the fan the value that I defined in Target GPU Temp. (℃): for example it is ignored.

I would like to work with both values, I define the value of the fan that I would like and if for some problem this temperature exceeds what I defined in Target GPU Temp. (℃): for example the fan increases the rotation or the mining stops for a few minutes
